src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -29,24 +29,26 @@ public class TypicalPersons {
 
     public static final Person CARL = new PatientBuilder().withNric("S2378593A").withName("Carl Sim")
             .withDoB("2005-01-03").withPhone("87436749").build();
-
-    // PATIENTS
-
```

Comment:
why did you delete the patients comment annotation?

Reply:
When I was resolving conflicts I made a mistake.